src/scales/scale.linear.js

```diff
@@ -84,10 +85,10 @@ module.exports = function(Chart) {
 
 							if (opts.relativePoints) {
 								positiveValues[index] = 100;
-							} else if (value < 0) {
-								negativeValues[index] += value;
+							} else if (value.val < 0) {
```

Comment:
It seems weird to check `val` here. Why not check `min` instead and then just remove `val`?

Reply:
done